Wangscape/Options.h

```diff
@@ -17,13 +20,20 @@ class Options
     typedef std::vector<Clique> CliqueList;
 
     std::string filename;
+    std::string directory;
```

Comment:
That's probably a good idea. Where would you want to put them, considering that the values are used by functions that have access to `Options` but not `OptionsManager`?